fe/fe-core/src/main/java/org/apache/doris/alter/RollupJobV2.java

```diff
@@ -150,6 +151,55 @@ public RollupJobV2(long jobId, long dbId, long tableId, String tableName, long t
         this.rollupIndexName = rollupIndexName;
 
         this.rollupSchema = rollupSchema;
+
+        Database db = Env.getCurrentInternalCatalog().getDbOrException(dbId,
+                s -> new AnalysisException("Databasee " + s + " does not exist"));
+
+        OlapTable tbl;
+        try {
+            tbl = (OlapTable) db.getTableOrMetaException(tableId, Table.TableType.OLAP);
+        } catch (MetaNotFoundException e) {
+            throw new AnalysisException(e.getMessage());
+        }
+
+        IntSupplier colUniqueIdSupplier = new IntSupplier() {
+            public int pendingMaxColUniqueId = tbl.getMaxColUniqueId();
+
+            @Override
+            public int getAsInt() {
+                pendingMaxColUniqueId++;
+                return pendingMaxColUniqueId;
+            }
```

Comment:
1、now in rollup, we don't support light schema change, this is unnecessary. 
2、in the end, we also need update tbl max unique id . for example in  SchemaChangeJobV2.onFinished() , we do like this:
        //update max column unique id
        int maxColUniqueId = tbl.getMaxColUniqueId();
        for (Column column : tbl.getFullSchema()) {
            if (column.getUniqueId() > maxColUniqueId) {
                maxColUniqueId = column.getUniqueId();
            }
        }
        tbl.setMaxColUniqueId(maxColUniqueId);


3、i will refact the light schema change feature, here you can just delete this code about generating col_unique_id